templates/gitlab-ci-template-cloud-run.yml

```diff
@@ -164,9 +146,16 @@ test variables:
 build:
   stage: build
   extends: .build_template
+  artifacts:
```

Comment:
I would prefer to define the `artifacts` definition in the template and not in the final job.